pl_examples/loop_examples/mnist_lite.py

```diff
@@ -0,0 +1,186 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+from typing import Any
+
+import torch
+import torch.nn.functional as F
+import torch.optim as optim
+import torchvision.transforms as T
+from torch.optim.lr_scheduler import StepLR
+from torchmetrics import Accuracy
+
+from pl_examples.basic_examples.mnist_datamodule import MNIST
+from pl_examples.basic_examples.mnist_examples.image_classifier_1_pytorch import Net
+from pytorch_lightning import seed_everything
+from pytorch_lightning.lite import LightningLite
+from pytorch_lightning.loops import Loop
+
+
+class TrainLoop(Loop):
+    def __init__(self, lite, args, model, optimizer, scheduler, dataloader):
```

Comment:
typing?

Reply:
I'd rather not add typing to the examples, it can get quite distracting

Reply:
Yes, agree.

Reply:
well that is one side, but my side as a user is that I have no idea what the inputs really are, just guessing bu name